gui/src/components/maintenance/auditlog.vue

```diff
@@ -255,8 +290,55 @@ export default class CortxAuditLog extends Vue {
       text: "Response Code",
```

Comment:
{ 
{
{
text_id:"timestamp"
text: "Timestamp",
      value: "timestamp",
      sortable: true,
      type: "time",
     show:"true/false",
    
    },

in your auditlog component code
check id is present in en.json if yes then display that value if not fallback to "text" field